msteams-platform/TOC.yml

```diff
@@ -416,110 +416,107 @@
-    href: bots/bot-features.md
-    displayName: Bot samples, advantages of bots, bot with Power Virtual Agents, Microsoft Bot Framework
-  - name: Bot activity handlers
-    href: bots/bot-basics.md
-    displayName: OnMessageActivityAsync, Bot logic, core Bot Framework handlers
-  - name: Bot conversations
+    href: bots/overview.md
+    displayName: AI bots, bot capability, 
+  - name: Let's build a bot
+    href: bots/build-a-bot.md
+    displayName: Bot samples, advantages of bots, Teams AI library, Microsoft Bot Framework, build a bot, Build AI bot, configure bot scope, update app manifest
+  - name: Build an AI bot
     items:
-    - name: Overview
-      href: bots/how-to/conversations/conversation-basics.md
-      displayName: chat, Messages sent between your Microsoft Teams bot and one or more user
-    - name: Messages in bot conversations
-      href: bots/how-to/conversations/conversation-messages.md
-      displayName: Picture messages, emojis, attribution for bot message, status code responses, app localization using bot and tab, form, soft delete, send messages from bot
-      items:
-      - name: Receive all conversation messages
-        href: bots/how-to/conversations/channel-messages-with-rsc.md
-        displayName: mention, send messages from bot, receive chat messages, bot messages, bots with RSC permissions
-    - name: Create a commands menu
-      href: bots/how-to/create-a-bot-commands-menu.md
-      displayName: compose
-    - name: Create prompt suggestions
-      href: bots/how-to/conversations/prompt-suggestions.md
-      displayName: prompt starter, suggested actions 
+    - name: What is Teams AI library?
+      href: bots/how-to/teams-conversational-ai/teams-conversation-ai-overview.md
+      displayName: OpenAI, Azure OpenAI, LLM, ChatGPT, prompt, Math tutor, Assistants API, AI bot, custom engine agent, RAG, Intents to actions, AI
+    - name: Quick start guide
+      href: bots/how-to/teams-conversational-ai/conversation-ai-quick-start.md
+      displayName: custom engine agent, AI bot
+    - name: Let's build an AI bot
+      href: bots/how-to/teams-conversational-ai/how-conversation-ai-get-started.md
+      displayName: custom engine agent, AI bot, Teams AI library, prompt manager, Action planner, action handler, feedback loop
+    - name: Understand AI bot concepts
+      href: bots/how-to/teams-conversational-ai/how-conversation-ai-core-capabilities.md
+      displayName: AI activity handlers, Adaptive Card for AI bot, Teams AI library, AI library, Teams AI library bot logic, OpenAI
+    - name: Build bot messages with AI-generated content
+      href: bots/how-to/bot-messages-ai-generated-content.md
+      displayName: AI label, citations, feedback buttons, sensitivity label, Powered by AI UX Kit  
+    - name: Build custom engine agent
+      items: 
+      - name: Custom engine agent for Microsoft 365 
+        href: /microsoft-365-copilot/extensibility/overview-custom-engine-agent
```

Comment:
Please make all pages that take to the Copilot extensibility docs in contextual links, so we don't lose readers off from this Teams doc set! Please refer this: https://review.learn.microsoft.com/en-us/help/platform/navigation-set-up-contextual-toc?branch=main